modules/gdscript/gdscript_byte_codegen.h

```diff
@@ -332,7 +332,12 @@ class GDScriptByteCodeGenerator : public GDScriptCodeGenerator {
 	}
 
 	void append(GDScriptFunction::Opcode p_code, int p_argument_count) {
```

Comment:
I believe we can remove the argument count here since it's not being used anymore.

Reply:
Yeah, I have to, but its used in some places that I was afraid to mess up and introduce some bug, so I will do after its well tested.

Reply:
Alright, done.